cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -107,15 +107,24 @@ Whether they are FileSets or another type of dataset is handled by the CDAP runt
 .. highlight:: console
 
 However, you do need to tell CDAP the relative paths of the input and output files. Currently,
-this is only possible by specifying them as runtime arguments when the MapReduce program is started::
+this is only possible by specifying them as runtime arguments when the MapReduce program is started:
 
-  curl -v <base-url>/apps/FileSetExample/mapreduce/WordCount/start -d '{ \
-      "dataset.lines.input.paths":  "monday/my.txt", \
-      "dataset.counts.output.path": "monday/counts.out" }'
+.. tabbed-parsed-literal::
+
+  $ curl -w"\n" -X POST "http://example.com:10000/v3/namespaces/default/apps/FileSetExample/mapreduce/WordCount/start" \
```

Comment:
Does it make sense for us to have non-required options in these? (`-w"\n"`) I know that it makes the output "cleaner" for the user, but I prefer not having optional things on the command line as it can lead to confusion.